calicoctl/calico_ctl/node.py

```diff
@@ -82,11 +83,16 @@
                             will be launched via the calico-node container,
                             whereas a value of "none" will not launch them at
                             all. [default: docker]
+  --backend=<BACKEND>       Specify which networking backend to use.
+                            Choices are "bird", "gobgp" or "none".
+                            When set to "none", Calico node run in policy
+                            only mode. [default: bird]
```

Comment:
@djosborne I'm sorry for the wrong fix. I removed it and added two commits after yours to address @gunjan5's comments. PTAL.